tests/DotNetLightning.Core.Tests/Serialization.fs

```diff
@@ -143,7 +143,7 @@ module SerializationTest =
                     let mutable expected = hex.DecodeData("d977cb9b53d93a6ff64bb5f1e158b4094b66e798fb12911168a3ccdf80a83096340a6a95da0ae8d9f776528eecdbb747eb6b545495a4319ed5378e35b21e073a1735b6a427e80d5fe7cd90a2f4ee08dc9c27cda7c35a4172e5d85b12c49d4232537e98f9b1f3c5e6989a8b9644e90e8918127680dbd0d4043510840fc0f1e11a216c280b5395a2546e7e4b2663e04f811622f15a4f91e83aa2e92ba2a573c139142c54ae63072a1ec1ee7dc0c04bde5c847806172aa05c92c22ae8e308d1d2692b12cc195ce0a2d1bda6a88befa19fa07f51caa75ce83837f28965600b8aacab0855ffb0e741ec5f7c41421e9829a9d48611c8c831f71be5ea73e66594977ffd")
                     expected <- Array.append expected (hex.DecodeData("0000"))
                     if nonbitcoinChainHash then
-                        expected <- Array.append expected (hex.DecodeData("43497fd7f826957108f4a30fd9cec3aeba79972084e90ead01ea330900000000"))
+                        expected <- Array.append expected (hex.DecodeData("000000000933ea01ad0ee984209779baaec3ced90fa3f408719526f8d77f4943"))

```

Comment:
nit: You can just `Array.rev`

Reply:
Is there a reason not to write it in that order though? This is the order that's expected on the wire, and the order that `uint256.Parse` accepts.

Reply:
For instance, line 148 (the other branch of this `if` expression) uses that order.

Reply:
Not really, yeah I've changed my mind, my concern was kind of bikeshedding.
 I will just merge as is.